### PR TITLE
LRQA-80219 | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/LRQA-80219

Tested here: https://github.com/Kvale1733/liferay-portal/pull/335
The one failure is not unique. All other tests pass.

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/bundleblacklist/BundleBlacklist.testcase [hunk 1]
```diff
@@ -20,48 +20,64 @@ definition {
...
+		}
 	}
 
 	@priority = 5
 	test BlacklistModuleBundleSymbolicName {
+		property osgi.modules.includes = "frontend-taglib-clay-sample-web";
 		property portal.acceptance = "true";
-		property test.name.skip.portal.instance = "BundleBlacklist#BlacklistModuleJar";
 
-		ApplicationsMenu.gotoPortlet(
```
Comment (kyle-miho) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/bundleblacklist/BundleBlacklist.testcase:30]: Keep the JSON Improvements though

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/fileinstall/OSGIConfiguration.testcase [hunk 2]
```diff
@@ -18,6 +18,13 @@ definition {
 		if (${testPortalInstance} == "true") {
 			PortalInstances.tearDownCP();
 		}
```
Comment (kyle-miho) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/fileinstall/OSGIConfiguration.testcase:20]: This is fine, but you should pull the cleanup outside of the else statement (testPortalInstance is set to true in CI runs or specific local runs, so this wont be cleaned up well in CI runs)

**Comment by kyle-miho** [comment]
@Kvale1733 see comments and resend please

**Pull request closed without merging**